cvmfs/CMakeLists.txt

```diff
@@ -365,7 +365,7 @@ if (BUILD_CVMFS)
 
   # link the stuff (*_LIBRARIES are dynamic link libraries *_archive are static link libraries ... one of them will be empty for each dependency)
   set (CVMFS_FUSE_LINK_LIBRARIES ${SQLITE3_LIBRARY} ${CARES_LIBRARIES}
-         ${CURL_LIBRARIES} ${LIBCURL_ARCHIVE} ${PACPARSER_LIBRARIES}
+         ${CURL_LIBRARIES} ${LIBCURL_ARCHIVE} ${NGHTTP2_ARCHIVE} ${PACPARSER_LIBRARIES}
```

Comment:
The nghttp2 library needs to be before the curl library/archive (like cares).  Otherwise linkers might not be able to resolve the symbols.